File: Array_Reverse.py
arr = [1, 2, 3, 4, 5, 6]
x = len(arr)
a = [0] * x
j = x - 1

for i in range(len(arr)):
    a[j] = arr[i]
    j -= 1

for i in range(x):
    print(f"{a[i]}\t", end="")

print()

File: Algo_C/Array_Reverse.c
#include<stdio.h>
int main(){
    int arr[]={1,2,3,4,5,6};
    int n=sizeof(arr)/sizeof(int);
    int a[n];
    int j=n-1;
    for (int i=0;i<sizeof(arr)/sizeof(int);i++){
        a[j]=arr[i];
        j--;
    }
    for (int i=0;i<n;i++){
        printf("%d\t",a[i]);
    }

    return 0;
}